test/OrchardCore.Tests/Apis/Context/SiteStartup.cs

```diff
@@ -17,6 +21,14 @@ static SiteStartup()
         public void ConfigureServices(IServiceCollection services)
 #pragma warning restore CA1822 // Mark members as static
         {
+            LogManager.Setup().SetupExtensions(delegate (ISetupExtensionsBuilder ext)
+            {
+                ext.RegisterLayoutRenderer<TenantLayoutRenderer>("orchard-tenant-name");
+            });
+            services.AddLogging(loggingBuilder =>
```

Comment:
Why not to use `UseNLogHost()`?

Reply:
I think the whole PR will be simplified :)

Reply:
Yes, you are correct, the PR has been updated